integration_tests/mockApis/placementRequests.ts

```diff
@@ -21,18 +21,21 @@ export default {
       },
     }),
 
-  stubPlacementRequestsDashboard: (placementRequests: Array<PlacementRequest>): SuperAgentRequest =>
+  stubPlacementRequestsDashboard: (args: {
+    placementRequests: Array<PlacementRequest>
+    isParole: boolean
+  }): SuperAgentRequest =>
     stubFor({
       request: {
         method: 'GET',
-        urlPattern: paths.placementRequests.dashboard.pattern,
+        url: `${paths.placementRequests.dashboard.pattern}?isParole=${args.isParole}`,
```

Comment:
Per the last commit I think we want this to be '1' if true, this will be a boolean